Day-1/main.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_instance" "app_server" {
+  ami           = "ami-06b09bfacae1453cb"
+  instance_type = "t2.micro"
+
+  tags = {
+    Name = "Task to deploy first simple AWS resource"
+  }
+}
```

Comment:
could you please, add newline at the end of file.

Reply:
Added!